lib/Modal/Modal.js

```diff
@@ -117,7 +117,7 @@ const Modal = (props) => {
     contentClass,
   );
 
-  const renderBackdrop = ({ onClick, ref, style }) => { /* eslint-disable-line react/prop-types */
+  const renderBackdrop = ({ onClick, ref, style }) => { /* eslint-disable-line react/prop-types, no-shadow */
```

Comment:
We shouldn't just disable eslint here. If we can't write this without shadowing, it's better to rename the `ref`, eg,

```suggestion
  const renderBackdrop = ({ onClick, ref: backdropRef, style }) => { /* eslint-disable-line react/prop-types, no-shadow */
```

I'm not saying use that name in particular, because honestly, I can't really follow who is providing that inner ref.